system/src/system_task.cpp

```diff
@@ -285,6 +286,16 @@ void establish_cloud_connection()
             return;
         }
 
+#if PLATFORM_ID==PLATFORM_ELECTRON_PRODUCTION
```

Comment:
I started there, but opted out because it seemed like more complexity at the time.  I'll look at refactoring it.